actions/download_action.go

```diff
@@ -1,3 +1,35 @@
+/*
+Download Action
+
+Download a single file from Internet and unpack it in-the-place if needed.
```

Comment:
… or you could say *in place*. In this that is, in fact, the correct way to say it: http://dictionary.cambridge.org/dictionary/english/place

Reply:
done